packages/tailwindcss-language-server/src/oxide.ts

```diff
@@ -58,10 +58,32 @@ declare namespace OxideV3 {
   }
 }
 
+// This covers the Oxide API from v4.1.0+
+declare namespace OxideV5 {
+  interface GlobEntry {
+    base: string
+    pattern: string
+    negated: boolean
+  }
+
+  interface ScannerOptions {
+    sources: Array<GlobEntry>
+  }
+
+  interface ScannerConstructor {
+    new (options: ScannerOptions): Scanner
+  }
+
+  interface Scanner {
+    files: Array<string>
+    globs: Array<GlobEntry>
```

Comment:
Not strictly necessary either, but matches the real generated types. This might actually result in errors if you are sort of polyfilling the `files` and `globs` array.
```suggestion
    get files(): Array<string>
    get globs(): Array<GlobEntry>
```